webapp/channels/src/sass/base/_structure.scss

```diff
@@ -87,6 +87,13 @@ body {
 }
 
 /* stylelint-disable -- grid-template indentation */
```

Comment:
This comment seems out of place, but the linter is not complaining. Let's leave it like this, unless anyone else has a stronger feeling about this.

Reply:
This wasn't added in this PR, so I'm inclined to leave it